### PR TITLE
Show Decisions in Portal ALC Tab

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

File: services/apps/alcs/src/portal/application-decision/application-decision.controller.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      ApplicationDecision,
+      PortalDecisionDto,
+    );
+  }
+
+  @Get('/:uuid/open')
+  async openFile(@Param('uuid') fileUuid: string, @Req() req) {
+    const url = await this.decisionService.getDownloadUrl(fileUuid);
+
+    //TODO: How do we know which documents applicant can access?
```
Comment (sandratoh) [inline services/apps/alcs/src/portal/application-decision/application-decision.controller.ts:45]: Is this for future ticket?

Comment (dhaselhan) [reply services/apps/alcs/src/portal/application-decision/application-decision.controller.ts:45]: Yes, kind of, we have a few of these left over from when we combined Portal into ALCS

File: portal-frontend/src/app/features/view-submission/alc-review/decisions/decisions.component.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  ngOnChanges(changes: SimpleChanges): void {
+    this.loadDecisions();
+  }
+
+  async openFile(uuid: string) {
+    const res = await this.decisionService.openFile(uuid);
+    if (res) {
+      window.open(res?.url, '_blank');
+    }
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply portal-frontend/src/app/features/view-submission/alc-review/decisions/decisions.component.ts:28]: Fixed!

**Review by sandratoh — APPROVED** [review]
🔥

**Pull request merged**